.idea/.gitignore

```diff
@@ -1,3 +1,4 @@
 # Default ignored files
 /shelf/
 /qaplug_profiles.xml
+/modules.xml
```

Comment:
Just checking. Is this change intended?
It doesn't refer to the modules.xml file that is added in this PR.

Reply:
Sorry, my fault. I didn't realise that it was a different gitignore file.

Reply:
But it doesn't seem to work. Maybe because the / in the beginning.

Reply:
But it might also be because the file is already committed into git. Once it's in git, it must be removed from git, by a special git command, for gitignore to work.

Reply:
Thanks for asking. This file is automatically generated by Ij but I do not see any sense in it appearing in other developer's setups. Might even mix up other versions. Will try to sort it.

Reply:
latest commit adds the file, looks OK by its contents